### PR TITLE
List pending requests UI

### PR DESCRIPTION
**Pull request by alex-chew**
*Description of changes:*
This PR introduces UI and tests for listing accounts which are pending Admin approval.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: dev-portal/src/pages/Admin/Accounts/PendingRequests.jsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+          <p>
+            Are you sure you want to deny this account request? The request will
+            be permanently deleted, and <strong>{account.emailAddress}</strong>{' '}
+            will need to sign up again to request an account.
+          </p>
+        </Modal.Content>
+        <Modal.Actions>
+          <Button onClick={onClose}>Cancel</Button>
+          <Button negative onClick={onConfirm}>
+            Promote
```
Comment (Trial-In-Error) [inline dev-portal/src/pages/Admin/Accounts/PendingRequests.jsx]: Should this read Promote?

File: dev-portal/src/services/accounts.js [hunk 2]
```diff
@@ -72,3 +80,33 @@ export const promoteAccountByIdentityPoolId = async identityPoolId => {
   }
   account.isAdmin = true
 }
+
+export const approveAccountRequestByIdentityPoolId = async identityPoolId => {
+  await resolveAfter(1500)
```
Comment (Trial-In-Error) [inline dev-portal/src/services/accounts.js]: Why the long-delayed resolves?

Comment (Trial-In-Error) [reply dev-portal/src/services/accounts.js]: Oh, I see. Mock data.

File: dev-portal/src/pages/Admin/Accounts/PendingRequests.jsx (hunk 1 above)
Comment (alex-chew) [reply dev-portal/src/pages/Admin/Accounts/PendingRequests.jsx]: Nope

**Pull request merged**